.github/workflows/benchmarking.yml

```diff
@@ -7,16 +7,16 @@ on:
   pull_request:
     paths:
       # test changes to Sentry SDK sources
-      - 'Sources/**'
```

Comment:
please split these kinds of lint diffs into separate PRs in the future

Reply:
I am actually going to do it for this PR too.

Reply:
https://github.com/getsentry/sentry-cocoa/pull/5072